pkg/ccl/backupccl/backup_processor.go

```diff
@@ -422,47 +398,22 @@ func runBackupProcessor(
 						// If this file reply has an inline SST, push it to the
 						// ch for the writer goroutine to handle. Otherwise, go
 						// ahead and record the file for progress reporting.
-						if len(file.SST) > 0 {
-							exportResponseTraceEvent.HasReturnedSSTs = true
-							ret := returnedSST{f: f, sst: file.SST, revStart: res.StartTime}
-							// If multiple files were returned for this span, only one -- the
-							// last -- should count as completing the requested span.
-							if i == len(res.Files)-1 {
-								ret.completedSpans = completedSpans
-							}
-							select {
-							case returnedSSTs <- ret:
-							case <-ctxDone:
-								return ctx.Err()
-							}
-						} else {
-							files = append(files, f)
+						exportResponseTraceEvent.HasReturnedSSTs = true
```

Comment:
Is this field still worth keeping around in the trace event if it's always true?